src/coreComponents/constitutive/contact/CoulombFriction.hpp

```diff
@@ -269,7 +280,7 @@ inline void CoulombFrictionUpdates::computeShearTraction( localIndex const k,
       dTractionVector_dJump[2][2] = m_shearStiffness;
 
       // The slip is only elastic: we add the full slip to the elastic one
-      LvArray::tensorOps::add< 2 >( m_elasticSlip[k], slip );
+      //LvArray::tensorOps::add< 2 >( m_elasticSlip[k], slip );
```

Comment:
@CusiniM this update and update on the lines 314-315 below got moved into `updateFractureState`.
The reason is: those quantities should not be updated at every Newton but once at the end of the time step insted.
`updateFractureState` is probably not the best place either, what would be the better one?

Reply:
@CusiniM could you please confirm that you agree with this change?

Reply:
That should be okay.